modules/openapi-generator/src/main/resources/Java/libraries/webclient/api.mustache

```diff
@@ -46,14 +46,81 @@ public class {{classname}} {
     }
 
     public ApiClient getApiClient() {
-        return apiClient;
+       return apiClient;
```

Comment:
Not important, but why remove the space here? It seemed good before.

Reply:
oh, that wasn't intentional